internal/graphicsdriver/metal/mtl/example_darwin_test.go

```diff
@@ -17,6 +17,7 @@ package mtl_test
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/ebitengine/purego"
```

Comment:
Move this to the same group as `"golang.org/x/image/math/f32"`

Reply:
Would using something like `gofumpt` help here? IIRC it automatically groups imports like this. Not sure if it matches existing Ebitengine code style though.

Reply:
I'll take a look later, thanks